docs/source/azure/gettingstarted.rst

```diff
@@ -50,12 +50,11 @@ To install Cloud Custodian, just run::
 Write your first policy
 -----------------------
 
-YAML Policies are the core concept of Cloud Custodian.  They take actions 
-on resources specified by filters. Custodian adheres to a compliance
-as code principle, so you can validate, dry-run, and code review on
-changes to your policies.
+The core concept is a stateless rule engine, that filters and takes actions on resources.
```

Comment:
Reads a little funny to me.  Maybe "Cloud Custodian is a stateless rules engine that filters and takes actions on resources."